xapian-applications/omega/mimemap.tokens

```diff
@@ -193,6 +193,13 @@ rpm		application/x-redhat-package-manager
 # Atom feeds:
 atom		application/atom+xml
 
+# Ebooks:
+fb2		application/x-fictionbook+xml
+fb2.zip		application/x-zip-compressed-fb2
+lrf		application/x-sony-bbeb
+tcr		application/x-tcr-ebook
+pdb		application/vnd.palm
+
```

Comment:
Should I associate `application/java-archive` to `jar`?

Reply:
Is `.jar` really an e-book format?  It's mainly a way to package up Java `.class` files.

`.jar` files are actually ZIP archives with a different extension and a particular layout inside, and ZIP containers are also quite popular for modern file formats, so I'm wondering if libmagic is getting confused and reporting `application/java-archive` for some e-book format.

Reply:
Oh I see, it's used by "QiOO".  It looks like that's an old format which never really caught on which actually shipped the Java software to read the book along with the book (I think I can see why that never caught on).

So probably better to leave it off the default list.  If someone has a collection of these they want to index, they can always run `omindex` with `-Mjar:application/java-archive` to override the default of ignoring `.jar` files.